src/wp-includes/media.php

```diff
@@ -6233,16 +6233,23 @@ function wp_high_priority_element_flag( $value = null ) {
  * @return string[] An array of mime type mappings.
  */
 function wp_get_image_editor_output_format( $filename, $mime_type ) {
+	$default_output_format = array(
+		'image/heic'          => 'image/jpeg',
+		'image/heif'          => 'image/jpeg',
+		'image/heic-sequence' => 'image/jpeg',
+		'image/heif-sequence' => 'image/jpeg',
+	);
+
 	/**
 	 * Filters the image editor output format mapping.
 	 *
-	 * Enables filtering the mime type used to save images. By default,
-	 * the mapping array is empty, so the mime type matches the source image.
+	 * Enables filtering the mime type used to save images. By default HEIC/HEIF images
+	 * are converted to JPEGs.
 	 *
 	 * @see WP_Image_Editor::get_output_format()
 	 *
 	 * @since 5.8.0
-	 * @since 6.7.0 The default was changed from array() to array( 'image/heic' => 'image/jpeg' ).
+	 * @since 6.7.0 The default was changed from empty array to array containing the HEIC mime types.
```

Comment:
```suggestion
	 * @since 6.7.0 The default was changed from empty array to an array containing the HEIC mime types.
```

Grammar feels wrong.

Reply:
Yea the grammar feels a bit wrong. If this is supposed to be a whole sentence shouldn't it be better as:
`@since 6.7.0 The default was changed from an empty array to an array containing the HEIC mime types.`?

Reply:
Fine for me.